chef/cookbooks/provisioner/templates/suse/crowbar_register.erb

```diff
@@ -110,8 +112,15 @@ add_user() {
   if [[ -z "$uid" ]]; then
     useradd --system --shell /sbin/nologin -d / --gid $2 --uid $2 --groups $3 $1
```

Comment:
With `--groups $3` this will create a user with group `openstack-glance` which is incorrect

Reply:
I don't follow. According to the man page `useradd --groups` doesn't create groups, only adds the user also to that group. For the glance case `$3` is `glance` and only `$4` is `openstack-glance`. Did I miss anything?

Reply:
You're right, I was looking at the add_group on line 196 and not at add_user. Sorry for the confusion.